javascript/chronometer.js

```diff
@@ -1,26 +1,71 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
+
   startClick(callback) {
     // ... your code goes here
+    
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (callback) {
+      callback();
+      }
+    }, 10)
   }
+
   getMinutes() {
     // ... your code goes here
+    return Math.floor((this.currentTime / 100) / 60);
   }
+
+
   getSeconds() {
     // ... your code goes here
+    return Math.floor((this.currentTime / 100) % 60);
+    
   }
-  twoDigitsNumber() {
+
+  getMiliSeconds() {
     // ... your code goes here
+    return Math.floor(this.currentTime % 100);
+    
   }
+
+  twoDigitsNumber(num) {
+    // ... your code goes here
+    
+    let n = num.toString();
```

Comment:
remember to use better naming for your variables, for this case could be numAsStr or something similar.